packages/editor/src/components/provider/index.js

```diff
@@ -323,7 +323,8 @@ export default compose( [
 			selectionEnd: getEditorSelectionEnd(),
 			reusableBlocks: select( 'core' ).getEntityRecords(
 				'postType',
-				'wp_block'
+				'wp_block',
+				{ 'per_page': -1 }
```

Comment:
There's a linting error—the quote aren't needed. Fix that and should be good to merge 👍 

Thanks for the fix, Paul, and congrats on the first contribution.

Reply:
Thanks, Dan. Sorry, I didn't have my linting setup correctly. It should be fixed now.